Sizzle/Sizzle/Sizzle/recipeView.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  recipeView.swift
+//  Sizzle
+//
+//  Created by Charlie Nevitt on 11/2/22.
+//
+
+import SwiftUI
+
+struct recipeView: View {
+    let image:String
+    let recipeName: String
+    @State private var selectedTab = "one"
```

Comment:
I know you're just messing around, but this is a great spot for an `enum` with some cases for your different tabs.